docker-compose.yml

```
@@ -0,0 +1,13 @@
+version: '3.9'
+
+services:
+  db:
+    build: .
```

If your intention is to build the database using your custom Dockerfile... You need to modify this to
```suggestion
    build:
        context: .
        dockerfile: Dockerfile
```

---

You also need to remove the line
image: postgres

---

Because what this script is doing now is:
1. When you run `docker compose up`
2. The docker is initializeing a service called db with the default image postgres, not your custom Dockerfile one. It will run your custom dockerfile IF locally you already build your custom Dockerfile image with the name postgres. But it will only work in your machine in that way.

---

Done